src/utils/spark.py

```diff
@@ -107,6 +116,19 @@ def __init__(
             .as_posix()
         )
 
+    def get_description(self) -> str:
+        """
+        Returns a formatted string describing the job, visible in the Spark UI.
+        """
+        desc = [f"{self.table_name}"]
+        if self.taxyr:
+            min_year, max_year = self.taxyr[0], self.taxyr[-1]
+            desc.append(f"taxyr=[{min_year}, {max_year}]")
+        if self.cur:
+            desc.append(f"cur=[{', '.join(self.cur)}]")
+
+        return ", ".join(desc)
+
```

Comment:
Just reordered this to be in alphabetical order with the other methods. No actual changes.